tests/components/Card.test.js

```diff
@@ -0,0 +1,32 @@
+import Card from '../../src/components/Card/Card'
+import { render } from '@testing-library/react'
+
+describe('<Card />', () => {
+  let mockCardData = {
+    created: '1900-01-01',
+    title: 'Test Card',
+    company: { display_name: 'Company XYZ' },
+    location: { display_name: 'Location' },
+    description: 'main card info',
+    redirect_url: 'http://example.com',
+  }
```

Comment:
Test another object with mock data testing `removeHtml` function. Also test sad data where data is malformed or missing